packages/vega/test/scenegraphs/area.json

```diff
@@ -473,6 +473,7 @@
               ],
               "x": 0.5,
               "y": 200.5,
+              "offset": 0,
```

Comment:
Can you assume that the default is 0 and keep the scenegraph leaner?

Reply:
Not sure. I've tried to find where offset is being set on group items but haven't found it yet. Alternatively, we could modify the JSON serialization method to exclude offset except when serializing gradients.